hoge.rb

```diff
@@ -0,0 +1,2 @@
+a =  aa
+%w[1 'a',]
```

Comment:
__[development][rubocop]__
* `%w`-literals should be delimited by `(` and `)`. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-literal-braces)

<!-- Comment by [SideCI](https://sideci.com) -->